src/internal/operations/restore_test.go

```diff
@@ -218,38 +222,68 @@ func (suite *RestoreOpIntegrationSuite) SetupSuite() {
 	require.NotEmpty(t, bo.Results.BackupID)
 
 	suite.exchange = bupResults{
-		backupID: bo.Results.BackupID,
-		// Discount metadata files (3 paths, 3 deltas) as
+		selectorResourceOwners: users,
+		resourceOwner:          m365UserID,
+		backupID:               bo.Results.BackupID,
+		// Discount metadata collection files (1 delta
+		// and one prev path for each category) as
 		// they are not part of the data restored.
 		items: bo.Results.ItemsWritten - 6,
 	}
 
-	siteID := tester.M365SiteID(t)
-	sites := []string{siteID}
-	csel := selectors.NewSharePointBackup(sites)
-	csel.DiscreteOwner = siteID
-	csel.Include(csel.LibraryFolders(selectors.Any()))
+	var (
+		siteID = tester.M365SiteID(t)
+		sites  = []string{siteID}
+		spsel  = selectors.NewSharePointBackup(sites)
+	)
+
+	spsel.DiscreteOwner = siteID
+	// assume a folder name "test" exists in the drive.
```

Comment:
I think we have a page somewhere that documents the minimum data set required to stand up CI on a new tenant. Can you add this there please?